packages/circuit-ui/components/List/List.tsx

```diff
@@ -109,7 +109,7 @@ const marginStyles = ({ noMargin }: ListProps) => {
   `;
 };
 
-const BaseList = styled('ol', {
+const BaseList = styled('ul', {
```

Comment:
This doesn't change anything since the element is determined based on the `variant`, but it aligns it with the default for clarity